src/gather/selector.rs

```diff
@@ -55,7 +55,7 @@ pub enum ParsedExpression {
     #[regex(r"\w+\s+notin\s+\([\w\s,]+\)", |lex| parse_set(lex.slice()))]
     #[regex(r"\!\w+", |lex| parse_set(lex.slice()))]
     #[regex(r"\w+", |lex| parse_set(lex.slice()))]
-    #[regex(r"\w+\s*=\s*\w+", |lex| parse_equality(lex.slice()))]
+    #[regex(r"[-./\w]+\s*=\s*[-.\w]+", |lex| parse_equality(lex.slice()))]
```

Comment:
Looks good, can you also add this regex to other occurrences of \w in other expressions?

Reply:
It does not seem to be needed, LGTM as is :)